autoscheduler/frontend/src/utils/createThrottleFunction.ts

```diff
@@ -27,7 +31,7 @@ const throttle = ((): (id: string, callback: any, delay: number, runOtherIDs: bo
         const toRun = functions.get(id);
         if (typeof toRun === 'function') toRun();
         functions.delete(id);
-        window.removeEventListener('beforeunload', functions.get(id));
+        window.removeEventListener('beforeunload', toRun);
```

Comment:
Nice, so before you fixed this, it used to attempt to get the function that you deleted in the line before?

Reply:
Correct